src/Peering/Peering/Direct/SetAzureDirectPeeringConnectionCommand.cs

```diff
@@ -235,13 +235,13 @@ private PSDirectConnection UpdateIpV4Prefix()
         /// </returns>
         private PSDirectConnection UpdateIpV6Prefix()
         {
-            if (this.InputObject is PSDirectConnection inputObject)
+            if (this.InputObject.GetType().Equals(typeof(PSDirectConnection)))
```

Comment:
Why use direct type comparsion instead of `is`? Except for extremely rare cases, `is` is what you want.